src/test/java/io/appform/dropwizard/actors/actor/metadata/MessageExpiredMetaGeneratorTest.java

```diff
@@ -0,0 +1,62 @@
+package io.appform.dropwizard.actors.actor.metadata;
+
+
+import static io.appform.dropwizard.actors.common.Constants.MESSAGE_EXPIRY_TEXT;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import io.appform.dropwizard.actors.actor.MessageMetadata;
+import java.time.Instant;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class MessageExpiredMetaGeneratorTest {
+
+    private MessageExpiredMetaGenerator generator;
+
+    @BeforeEach
+    void setup() {
+        generator = new MessageExpiredMetaGenerator();
```

Comment:
What is the need for creating class level variable when this can just be a local variable. Remove this and fix in all relevant tests